.binaries

```diff
@@ -1,4 +1,3 @@
-terraform-docs  0.9.1   https://github.com/segmentio/terraform-docs/releases/download/v{version}/terraform-docs-v{version}-{platform}-amd64
-packer          1.5.4   https://releases.hashicorp.com/packer/{version}/packer_{version}_{platform}_amd64.zip
-terraform-12    0.12.29 https://releases.hashicorp.com/terraform/{version}/terraform_{version}_{platform}_amd64.zip|terraform
-terraform-13    0.13.0 https://releases.hashicorp.com/terraform/{version}/terraform_{version}_{platform}_amd64.zip|terraform
```

Comment:
The earliest version that worked with all the modules was 0.15 (in this version they changed how provider aliases are forward into modules). Since Terraform doesn't assure compatibility with 0.15 but only with 1.0, we decided to go directly to "only 1.0". 
It's going to be much easier to maintain